tests/integration/states/test_file.py

```diff
@@ -2137,62 +2137,61 @@ def test_issue_11003_immutable_lazy_proxy_sum(self):
     @skipIf(six.PY3, 'This test will have a LOT of rewriting to support both Py2 and Py3')
```

Comment:
Should work under PY3 and Windows (esp if PY3.6.3)

Reply:
Yes, I meant to remove the other `skipIf` decorators and did not. Will un-skip these.